engine/src/main/java/org/hibernate/validator/internal/util/logging/Log.java

```diff
@@ -233,11 +233,16 @@ UnexpectedTypeException getNoValidatorFoundForTypeException(String constraintTyp
 	@Message(id = 64, value = "Unable to instantiate %1$s: %2$s.")
 	ValidationException getUnableToInstantiateException(String message, Class<?> clazz, @Cause Exception e);
 
+	@Deprecated
 	@Message(id = 65, value = "Unable to load class: %s.")
 	ValidationException getUnableToLoadClassException(String className);
 
+	@Deprecated
 	ValidationException getUnableToLoadClassException(String className, @Cause Exception e);
 
+	@Message(id = 65, value = "Unable to load class: %s from %s.")
```

Comment:
We normally don't re-use ids, but I guess in this case one could argue that it is still the same exception, just with improved context. We also just remove unused log messages. No need to deprecate. I'll care of these things, since I want to squash the commits in the end anyways.